### PR TITLE
Adding home page skeleton and navbar

### PR DESCRIPTION
**Pull request by fangyu3**

### CODE REVIEW

File: client/src/pages/Home.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                      classes={{ indicator: classes.tabIndicator }} >
+                  <Tab className={classes.tab} label={<Typography variant="subtitle1">EVENT TYPES</Typography>} />
+                  <Tab className={classes.tab} label={<Typography variant="subtitle1">SCHEDULED EVENTS</Typography>} />
+                </Tabs>
+            </AppBar>
+            </Grid>
+        </Grid>
+
+        {value===0 && <div>Event page</div>}
+        {value===1 && <div>Appointment page</div>}
```
Comment (johnsciutto) [inline client/src/pages/Home.js]: Works for now, but it's not sustainable to use arbitrary numbers to determine which component to render. It makes for confusing code really fast. Creating routes with `react-router-dom` might be a more sustainable choice.

Comment (sina-jamshidi) [reply client/src/pages/Home.js]: Actually yes Tony if it is possible to use routing here that would be great!
If you try and it's not simple to implement, you can just go back to this approach, but try using the `TabPanel` approach that is in the MaterialUI documentation under the Tabs component. We can discuss this more in standup

File: server/.env [hunk 2]
```diff
@@ -1,2 +1,2 @@
-TEAM_NAME="Shums,Tony Fang"
```
Comment (sina-jamshidi) [inline server/.env]: we can remove the `TEAM_NAME` variable entirely it's not necessary. Also make sure you gitignore your `.env`

**Review by sina-jamshidi — APPROVED** [review]

File: client/src/pages/Home.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    tab: {
+        marginRight:"25px",
+        paddingLeft:"0px",
+        color:theme.palette.secondary.main,
+    }
+
+}));
+
+
+function TabPanel(props) {
```
Comment (fangyu3) [inline client/src/pages/Home.js:40]: @sina-jamshidi Added the tab panels

File: client/src/App.js [hunk 4]
```diff
@@ -1,17 +1,20 @@
...
 
 import "./App.css";
 
 function App() {
   return (
     <MuiThemeProvider theme={theme}>
       <BrowserRouter>
-        <Route path="/" component={LandingPage} />
+        <Header/>
+        <Route exact path={["/", "/home", "/home/:page"]} render={(props) => <Home {...props} />} />
```
Comment (fangyu3) [inline client/src/App.js:17]: @sina-jamshidi I added in react router for the tabs, and the url changes when I click on them. Unfortunately, I was not able to get it working with the browser back button. I think we can leave as it is for now?

Comment (sina-jamshidi) [reply client/src/App.js:17]: Okay let's leave it for now

**Pull request merged**